templates/overrides/search/form-element--oe-whitelabel-search-form.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
+/**
+ * @file
+ * Theme implementation for the search form element.
+ *
+ * Display search form without wrappers or other elements like label, prefix, suffix or description.
+ *
+ * @see ./core/modules/system/templates/form-element.html.twig
+ */
+#}
+{% set classes = [
+  'js-form-item',
+  'form-item',
+  'js-form-type-' ~ type|clean_class,
+  'form-item-' ~ name|clean_class,
+  'js-form-item-' ~ name|clean_class,
+  title_display not in ['after', 'before'] ? 'form-no-label',
+  disabled == 'disabled' ? 'form-disabled',
+  errors ? 'form-item--error',
+] %}
```

Comment:
All of these classes are now only used in the `if` part below, and even then they are questionable.